### DISCUSSION TITLE
Acceso invalido

### DISCUSSION THREAD

**Discussion by Clara-Lopez** [community/community#138920] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

¿Cuál es el motivo por el que cuando intento ingresar sale lo siguiente?
El acceso a su cuenta ha sido suspendido debido a una infracción de nuestros
siendo que yo no incumpli ninguna regla ¿y como hago para que mi cuenta deje de estar suspendida?

**Comment by ebndev** [comment]
Hi @Clara-Lopez, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
